firefox-ios/Client/Frontend/Browser/Tabs/Views/TabDisplayDiffableDataSource.swift

```diff
@@ -6,8 +6,8 @@ typealias TabDisplayViewSection = TabDisplayDiffableDataSource.TabSection
 typealias TabDisplayViewItem = TabDisplayDiffableDataSource.TabItem
 
 final class TabDisplayDiffableDataSource: UICollectionViewDiffableDataSource<TabDisplayViewSection, TabDisplayViewItem> {
-    enum TabSection: Int, CaseIterable {
-        case inactiveTabs
+    enum TabSection: Hashable {
+        case inactiveTabs(UUID)
         case tabs
     }
```

Comment:
this feels a little hacky still, but better than before I think. I'm open to suggestions though @cyndichin

Reply:
thanks @dataports! making sections `Hashable` makes sense but the `UUID` does seem like a hack. I actually prefer the previous solution since it was less changes and can be isolated out of the updateSnapshot in general.

Reply:
The previous solution unfortunately introduced a bug to fix this, I'll update you when we meet next week so hopefully that will clarify why I did this this way and not the previous way.